controller/main.go

```diff
@@ -81,32 +83,32 @@ func (c *controller) SetBalancer(l log.Logger, name string, svcRo *v1.Service, _
 		st, svc = svc.Status, svcRo.DeepCopy()
 		svc.Status = st
 		if err := c.client.UpdateStatus(svc); err != nil {
-			l.Log("op", "updateServiceStatus", "error", err, "msg", "failed to update service status")
+			level.Error(l).Log("op", "updateServiceStatus", "error", err, "msg", "failed to update service status")
 			return k8s.SyncStateError
 		}
 	}
-	l.Log("event", "serviceUpdated", "msg", "updated service object")
+	level.Info(l).Log("event", "serviceUpdated", "msg", "updated service object")
 
 	return k8s.SyncStateSuccess
 }
 
 func (c *controller) deleteBalancer(l log.Logger, name string) {
 	if c.ips.Unassign(name) {
-		l.Log("event", "serviceDeleted", "msg", "service deleted")
+		level.Info(l).Log("event", "serviceDeleted", "msg", "service deleted")
```

Comment:
This seems like a debug message to me.

Unrelated to your change, this message also could be misleading. MetalLB didn't delete the service. We're just emitting a debug message saying that we saw a service get deleted. The text could be improved ...